src/assets/wise5/components/common/feedbackRule/feedback-rule-help/feedback-rule-help.component.html

```diff
@@ -111,6 +111,27 @@ <h3 i18n>Operators</h3>
       </td>
     </tr>
   </table>
+  <h3 i18n>Parentheses</h3>
+  <p i18n>Parentheses let you group expressions and prioritize evaluation.</p>
+  <p i18n>You can add multiple parentheses in an expression. You can even next parentheses.</p>
```

Comment:
Looks like there's a typo here. What is this sentence supposed to say: "You can even next parentheses."?

Reply:
Ah I meant to type "You can even nest parentheses."